packages/moonstone/ExpandableInput/ExpandableInput.module.less

```diff
@@ -1,9 +1,10 @@
 // ExpandableInput.module.less
 //
+@import "../styles/variables.less";
 
 .expandableInput {
 	.decorator {
 		display: flex;
-		margin: 6px;
+		margin: @moon-spotlight-outset;
```

Comment:
Let's set this to be an input variable: `@moon-expandableinput-input-margin`.
And define that as the following in variables.less:
```
@moon-expandableinput-input-margin: 6px @moon-spotlight-outset;
```
This way, we maintain the existing vertical sizing, and we update to the standard horizontal spacing, while using only component-variables in each of the component modules files.